cmd/gentypes/test/fullmsgpackexttype.go

```diff
@@ -0,0 +1,59 @@
+// Package test contains testing types and scenarios.
+package test
+
+import (
+	"bytes"
+
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+// FullMsgpackExtType is a test type with both MarshalMsgpack and UnmarshalMsgpack methods.
+type FullMsgpackExtType struct {
+	A int
+	B string
+}
+
+// NewEmptyFullMsgpackExtType is an empty constructor for FullMsgpackExtType.
+func NewEmptyFullMsgpackExtType() (out FullMsgpackExtType) { //nolint:nonamedreturns
+	return
+}
+
+// MarshalMsgpack .
```

Comment:
A comment is missed?

Reply:
It's test type, I've thought it's not needed to be documented at all.